### PR TITLE
feat(genapi): getting ready for public beta

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

- Rate limits defined
- 429 errors explained
- New models added
- Playground explained

### CODE REVIEW

File: ai-data/generative-apis/quickstart.mdx [hunk 1]
```diff
@@ -32,7 +32,17 @@ Hosted in European data centers and priced competitively per million tokens used
 
 ## Start with the Generative APIs Playground
 
-Scaleway's Playground is in development, stay tuned!
+Scaleway provides a web playground for instruct-based models hosted on Generative APIs.
+
```
Comment (bene2k1) [inline ai-data/generative-apis/quickstart.mdx:36]:
```suggestion

### Accessing the Playground
```

File: ai-data/generative-apis/quickstart.mdx [hunk 2]
```diff
@@ -32,7 +32,17 @@ Hosted in European data centers and priced competitively per million tokens used
...
+1. Navigate to Generative APIs under the AI section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
+2. Click the name of the chat model you want to try. Alternatively, click <Icon name="more" /> next to the chat model, and click **Try model** in the menu. 
+
+The web playground displays.
+
```
Comment (bene2k1) [inline ai-data/generative-apis/quickstart.mdx:41]:
```suggestion

### Using the Playground
```

File: ai-data/generative-apis/how-to/query-text-models.mdx [hunk 3]
```diff
@@ -27,7 +27,17 @@ There are several ways to interact with text models:
...
 ## Accessing the Playground
 
-Scaleway's Playground is in development, stay tuned!
+Scaleway provides a web playground for instruct-based models hosted on Generative APIs.
+
+1. Navigate to Generative APIs under the AI section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
+2. Click the name of the chat model you want to try. Alternatively, click <Icon name="more" /> next to the chat model, and click **Try model** in the menu. 
+
+The web playground displays.
+
```
Comment (bene2k1) [inline ai-data/generative-apis/how-to/query-text-models.mdx:36]:
```suggestion

## Using the Playground
```

**Review by bene2k1 — APPROVED** [review]

File: ai-data/generative-apis/api-cli/understanding-errors.mdx [hunk 4]
```diff
@@ -32,6 +32,8 @@ Below are usual HTTP error codes:
 - 404 - **Route Not Found**: The requested resource could not be found. Check your request is being made to the correct endpoint.
 - 422 - **Model Not Found**: The `model` key is present in the request payload, but the corresponding model is not found.
 - 422 - **Missing Model**:  The `model` key is missing from the request payload.
+- 429 - **Too Many Requests**: You are exceeding your current quota for the requested model, calculated in requests per minute. Find rate limits in [this page](/ai-data/generative-apis/reference-content/rate-limits/)
```
Comment (RoRoJ) [inline ai-data/generative-apis/api-cli/understanding-errors.mdx:35]:
```suggestion
- 429 - **Too Many Requests**: You are exceeding your current quota for the requested model, calculated in requests per minute. Find rate limits on [this page](/ai-data/generative-apis/reference-content/rate-limits/)
```

File: ai-data/generative-apis/api-cli/understanding-errors.mdx [hunk 5]
```diff
@@ -32,6 +32,8 @@ Below are usual HTTP error codes:
...
+- 429 - **Too Many Tokens**: You are exceeding your current quota for the requested model, calculated in tokens per minute. Find rate limits in [this page](/ai-data/generative-apis/reference-content/rate-limits/)
```
Comment (RoRoJ) [inline ai-data/generative-apis/api-cli/understanding-errors.mdx:36]:
```suggestion
- 429 - **Too Many Tokens**: You are exceeding your current quota for the requested model, calculated in tokens per minute. Find rate limits on [this page](/ai-data/generative-apis/reference-content/rate-limits/)
```

**Review by ofranc — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Comment by tgenaitay** [comment]
Oopsie I have rebased to main by accident. Closing this to do a clean PR

**Comment by tgenaitay** [comment]
https://github.com/scaleway/docs-content/pull/3923

**Pull request closed without merging**